Features/CopyMoveDelete/CopyContent.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Copy content
+    In order to copy objects
+    As an Editor user
+    I need to be able to copy an object that I am viewing
+
+    Background:
+        Given I am logged in as an Editor in PlatformUI
+
+    ##
+    #Copy objects to valid locations
+    ##
+    @javascript
+    Scenario: Copy one object without children objects
+        Given a "Destiny" folder exists
+        And a "News Flash" article exists
+        And I am viewing the "News Flash" article
+        When I click on "Copy" link on the "Action Bar"
+        And I select the "Destiny" folder in Universal Discovery Widget
+        And I click the "Confirm selection" button
+        Then I see the message "'News Flash' has been successfully copied under 'Destiny'"
```

Comment:
But then again, we have the translations...
So that function you referred would be an incomplete solution. We would have to have some king of mapping or dictionary for **every** sentence/button/option/notification subject to change with translations.  
```PHP
(...)
$notificationMessage = sprintf(
    "'%s%s%s'",
    $contentName, $MessageTranslatedToLangXPTO, $destinationName
  );
(...)
```
This also applies for the validation of "Copy", "Move", "Send to Trash", etc.

Translations: Is that really the path we want to follow now?